app/src/main/res/drawable/scheduled_message_icon.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:width="@dimen/small_icon_size"
+        android:height="@dimen/small_icon_size"
+        android:drawable="@drawable/ic_clock_vector" />
+
+</layer-list>
```

Comment:
do we really need this new file? Cant we just use the vector directly and set the size somehow else?

Reply:
sure, we can use `Drawable.setBounds()` to set the size in code.